### DISCUSSION TITLE
Run the embedded-cassandra

### DISCUSSION THREAD

**Discussion by c3-YufengSu** [nosan/embedded-cassandra#400] [Q&A]
Hi,
Is this project a Cassandra connector only? Or does it contain a complete Cassandra inside thus I can start it directly without having a Cassandra instance running in my system?

**Comment by nosan** [comment]
Hi. It is not a Cassandra connector. This project gives you the functionality to launch Cassandra. 

>  Cassandra inside thus I can start it directly without having a Cassandra instance running in my system?

Exactly
